terraform/modules/emr/templates/emr/cloudwatch.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+cwa_metrics_collection_interval="$1"
+cwa_log_group_name="$2"
+aws_default_region="$3"
+cwa_namespace="$4"
+
+# Create config file required for CloudWatch Agent
+mkdir -p /opt/aws/amazon-cloudwatch-agent/etc
+cat > /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json <<CWAGENTCONFIG
+{
+  "agent": {
+    "metrics_collection_interval": $${cwa_metrics_collection_interval},
+    "logfile": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log"
+  },
+  "logs": {
+    "logs_collected": {
+      "files": {
+        "collect_list": [  
+          {
+            "file_path": "/var/log/get_scripts.log",
+            "log_group_name": "$${cwa_log_group_name}",
+            "log_stream_name": "get_scripts.log",
+            "timezone": "UTC"
+          }
+          
+        ]
+      }
+    },
+    "log_stream_name": "$${cwa_namespace}",
+    "force_flush_interval" : 15
+  }
+}
+CWAGENTCONFIG
+
+%{ if emr_release == "5.29.0" ~}
```

Comment:
We can simplify and remove the if statement, as we are using 5.3.X so just keep the 'else' part.